blockware/matrix/lib/Touch/Touch.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// Accelerometer
+#include <LIS2DW12Sensor.h>
+
+class Touch {
+private:
+  static const uint8_t axes_count = 3;
+  static const uint8_t axes_history_length = 15;
+  static const int16_t DEFAULT_SENSITIVITY = 25;
+
+  bool debug = false;
```

Comment:
Minor: in general, unless `debug` will be toggled dynamically at runtime somehow, it's better to do `#define`/`#ifdef` for debugging functionality. Then you don't incur the runtime cost of checking the `debug` variable.

Reply:
oh right that’s exactly what the preprocessors are meant to do good point!!